core/src/components/progress-bar/readme.md

```diff
@@ -0,0 +1,70 @@
+# ion-progress-bar
+
+ion-progress-bar is a horizontal progress bar to visualize the progression of an operation and activity. You can choose between two types: ``determinate`` and ``indeterminate``.
+
+## Progress modes
+
+### determinate (default)
```

Comment:
Can we capitalize the headers here

```
### Determinate
### Indeterminate
```

And then remove the `(default)` in the header in favor of saying it in the paragraph below. It will just make it easier to link to this in the docs like: `ionicframework.com/docs/api/progress-bar#determinate`